workflow/usage.rst

```diff
@@ -450,3 +459,220 @@ The following example shows these functions in action:
+                        <framework:to>review</framework:to>
+                        <framework:metadata>
+                            <framework:priority>0.5</framework:priority>
+                        </framework:metadata>
+                    </framework:transition>
+                    <!-- ... -->
+                </framework:workflow>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/workflow.php
+
+        $container->loadFromExtension('framework', array(
+            // ...
+            'workflows' => array(
+                'blog_publishing' => array(
+                    'metadata' => array(
+                        'title' => 'Blog Publishing Workflow',
+                    ),
+                    // ...
+                    'places' => array(
+                        'draft' => array(
+                            'metadata' => array(
+                                'max_num_of_words' => 500,
+                            ),
+                        ),
+                        // ...
+                    ),
+                    'transitions' => array(
+                        'to_review' => array(
+                            'from' => 'draft',
+                            'to' => 'review',
+                            'metadata' => array(
+                                'priority' => 0.5,
+                            ),
+                         ),
+                     ),
+                 ),
+             ),
+         ));
+
+Then you can access this metadata in your controller as follows::
+
+    public function myControllerAction(Registry $registry, Article $article)
+    {
+        $workflow = $registry->get($article);
+
+        $workflow
+            ->getMetadataStore()
+            ->getWorkflowMetadata()['title'] ?? false
+        ;
+
+        // or
+        $workflow
+            ->getMetadataStore()
+            ->getPlaceMetadata('draft')['title'] ?? false
+        ;
+
+        // or
+        $aTransition = $workflow->getDefinition()->getTransitions()[0];
+        $workflow
+            ->getMetadataStore()
+            ->getTransitionMetadata($aTransition)['title'] ?? false
+        ;
+    }
+
+There is a shortcut that works with everything::
+
+    $workflow
+        ->getMetadataStore()
+        ->getMetadata('title')
+    ;
+
+In a Flash message in your Controller::
+
+            // $transition = ...; (an instance of Transition)
+            // $workflow is a Workflow instance retrieved from the Registry (see above)
+            $title = $workflow->getMetadataStore()->getMetadata('title', $transition);
+            $this->addFlash('info', "You have successfully applied the transition with title: '$title'");
+
+In a listener, access via the Event
+
+Metadata can also be accessed in a Listener, from the Event object.
+
+The example below uses a new feature introduced in 4.1 called Transition Blockers. These let you
```

Comment:
You could say here "Using transition blockers you can return user-friendly...", and add a note below the example like:

```rst
.. versionadded:: 4.1

    The transition blockers were added in version 4.1.
```

Reply:
I've also created another PR to document transition blockers: https://github.com/symfony/symfony-docs/pull/11212